### ISSUE TITLE
Can not remove some items from armory cabinet

### ISSUE THREAD

**Issue by SamancKing** [FTBTeam/FTB-Modpack-Issues#3148]
### Modpack

FTB Skies

### Modpack version

1.0.6

### Describe the bug

Can Not Remove gems form armory cabinet

### Steps to reproduce

first im using Storage scanner to acces armory cabinet
then when im trying to take the gems its not working
i can remove other items 

### Expected behaviour

i should be able to take it out

### Screenshots

![image](https://user-images.githubusercontent.com/130923685/232328875-604695cf-8e3d-4edc-9523-31f6d8fd45a5.png)

**Comment by afolk2** [comment]
I'm not familiar with the storage scanner, but the Armoury Cabinet says that you can't take items from it, it needs to be automated.

**Comment by SamancKing** [comment]
> I'm not familiar with the storage scanner, but the Armoury Cabinet says that you can't take items from it, it needs to be automated.

I Can take all the items out whit storage scanner only the gems i cant

**Comment by Saereth** [comment]
yeah you're not supposed to be able to take any items without automation so this is more a bug that you can take any. I tried piping gems in and out and its working fine
